tests/pages/owner/non-existent.js

```diff
@@ -0,0 +1,15 @@
+import {
+  create,
+  contains,
+  visitable,
+  text,
+  isHidden
+} from 'ember-cli-page-object';
+
+export default create({
+  visit: visitable('/non-existent-owner'),
+  showsBarricadeIllustration: contains('svg', { scope: '.page-graphic' }),
+  errorMessage: text('.missing-notice .page-title'),
+  errorMessageProisHidden: isHidden('.missing-notice p'),
+  errorMessageProUnauthenticated: contains('.missing-notice p')
```

Comment:
ahhhh now that I look at the actual page object, my previous commentary on the acceptance test isn’t quite adequate, because I didn’t understand the structure. This could look differently:

```
errorMessage: {
  scope: '.missing-notice',
  text: text('.page-title'),
  comMessage: {
    scope: 'p'
  }
}
```

Then you get `errorMessage.text`, `errorMessage.comMessage.isHidden` and `….isVisible`. What do you think?

Reply:
That's a good point. I want to get it out today, though, so I'll address this (and `repo-not-found` test where I copied it from) in a separate PR later this week.